OpenTrons/opentrons-api
# fix(components): fix cursor state on `EmptySelectorButton`

# Overview

When hovering an enabled `EmptySelectorButton`, (ex: walking through creating a protocol in PD redesign and adding pipettes/modules), the cursor should be `pointer`.

Closes RQA-3197

## Test Plan and Hands on Testing

- turn on PD redesign
- create a new protocol 
- walk through steps and verify that at steps 2 and 4, hovering the active `EmptySelectorButton` for "Add a pipette and tips" and individual module additions produces a pointer cursor

## Changelog

add cursor style to `EmptySelectorButton`

## Review requests

see test plan

## Risk assessment

low
